src/test/java/com/pwn9/filter/MockPlayer.java

```diff
@@ -40,13 +44,15 @@
 import org.bukkit.potion.PotionEffectType;
 import org.bukkit.scoreboard.Scoreboard;
 import org.bukkit.util.Vector;
+import org.jetbrains.annotations.NotNull;
 
 import java.net.InetSocketAddress;
 import java.util.*;
 
-public class MockPlayer implements Player {
+public class MockPlayer implements Player, com.pwn9.filter.engine.api.Player, CommandSender {
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'com.pwn9.filter.engine.api.Player' due to existing same package import 'com.pwn9.filter.*'](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)